Hunting Queries/AzureActivity/AzureNSG_AdministrativeOperations

```diff
@@ -0,0 +1,30 @@
+id: e94d6756-981c-4f02-9a81-d006d80c8b41
+name: Azure Network Security Group NSG Administrative Operations
+description: |
+  'Identifies set of Azure NSG administrative operational detection queries for hunting activites'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AzureActivity
+    dataTypes:
+      - AzureActivity
+queryFrequency: 1d
+queryPeriod: 7d
```

Comment:
Did you mean to make these Detections?  queryFrequency, queryPeriod, triggerOperator, triggerThreshold are only used for Detections per the [Query Style Guide](https://github.com/Azure/Azure-Sentinel/wiki/Query-Style-Guide).